src/main/sensors/barometer.c

```diff
@@ -343,6 +344,7 @@ bool baroIsCalibrated(void)
 
 static void baroSetCalibrationCycles(uint16_t calibrationCyclesRequired)
 {
+    // cycleCount = 0;
```

Comment:
```suggestion
    cycleCount = 0;
```
?

Reply:
Yes was not sure if we need to reset it here so made it a comment for now. Will activate the assignment.

Reply:
You can also set it to zero together with altitude, which is probably cleaner. And remove baroSetCalibrationCycles, is is a bit of overkill.

Or reverse the logic and set cycleCount, altitude and calibrationCyclesRequired in one function

Reply:
Implemented.